plugins/woocommerce/tests/e2e-pw/tests/merchant/create-variable-product.spec.js

```diff
@@ -94,6 +94,9 @@ test.describe.serial( 'Add New Variable Product Page', () => {
 		}
 
 		await page.locator( '#save-post' ).click();
+		await expect( page.locator( '#message.notice-success' ) ).toContainText(
+			'Product draft updated.'
+		);
```

Comment:
I added an assertion here to make sure that the product was successfully because the next test will use this product.